x-pack/plugins/lens/server/migrations/saved_object_migrations.ts

```diff
@@ -575,7 +583,7 @@ const lensMigrations: SavedObjectMigrationMap = {
   ),
   '8.3.0': flow(lockOldMetricVisSettings, preserveOldLegendSizeDefault, fixValueLabelsInXY),
   '8.5.0': flow(migrateMetricIds, enrichAnnotationLayers, migratePartitionChartGroups),
-  '8.6.0': flow(migrateIndexPatternDatasource),
+  '8.6.0': flow(migratePartitionMetrics, migrateIndexPatternDatasource),
```

Comment:
nit: Small thing, but AFAIK flow applies functions from left to right, so this means the partition migration is applied before the index pattern datasource migration, which is just opposite of what the embeddable migration is doing. It should be commutative in this case, but definitely a source of subtle weirdnesses.

Reply:
Good point, @flash1293 👍